packages/core/protocols/src/proto/dxos/halo/invitations.proto

```diff
@@ -6,13 +6,24 @@ syntax = "proto3";
 
 import "google/protobuf/empty.proto";
 
+import "dxos/client/services.proto";
+import "dxos/echo/timeframe.proto";
 import "dxos/halo/credentials.proto";
 import "dxos/keys.proto";
-import "dxos/echo/timeframe.proto";
 
 package dxos.halo.invitations;
 
-// TODO(burdon): Reconcile with client.services.Invitation. Merge files.
+message IntroductionRequest {
+  /// Guest's profile.
+  optional halo.credentials.ProfileDocument profile = 1;
+}
+
+message IntroductionResponse {
+  /// Only present for testing invitation types.
+  optional dxos.keys.PublicKey space_key = 1;
```

Comment:
I would swap those around (space_key is specific to space kind while auth method is generic invitation data):

auth_method = 1

space_key = 100 (leave space for more generic invitation stuff